scipy/stats/tests/test_multivariate.py

```diff
@@ -513,6 +513,15 @@ def test_frozen(self):
         assert_allclose(norm_frozen.cdf(x), multivariate_normal.cdf(x, mean, cov))
         assert_allclose(norm_frozen.logcdf(x),
                         multivariate_normal.logcdf(x, mean, cov))
+    
+    def test_frozen_multivariate_normal_exposes_attributes(self):
+        np.random.seed(1234)
```

Comment:
Some reviewers have a strong preference for only using NumPy `Generator`s, even in tests. I don't mind it, so to please everyone, this should be `rng = np.random.default_rng(349582469826092)` and `rng` should be passed into `multivariate_normal`.

Reply:
Leaving this was actually just an oversight on my part, the rest of the test is deterministic so the seeding can just be removed!

Reply:
Solved in [0e82be9](https://github.com/scipy/scipy/pull/17904/commits/0e82be9c6875ae18f647c8589f8eeee0080d37b6)